dask/dataframe/io/tests/test_json.py

```diff
@@ -10,6 +10,11 @@
     except ImportError:
         lzma = None
 
+try:
```

Comment:
Do we need a `try/except` for `json`? I think it's been included in the standard library since Python 2.6